apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ReportEventsTo.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api;
+using Google.Api.Gax;
+using Google.Cloud.ErrorReporting.V1Beta1;
+using Google.Cloud.Logging.V2;
+
+namespace Google.Cloud.Diagnostics.Common
+{
+    /// <summary>
+    /// The location error events will be sent to.
+    /// </summary>
+    public enum ReportEventsToLocation
+    {
+        /// <summary>Stackdriver Error Reporting API.</summary>
+        ErrorReporting,
+
+        /// <summary>Stackdriver Error Logging API.</summary>
+        Logging
+    }
+
+    /// <summary>
+    /// Represents the location error events will be sent, such as the Stackdriver Logging or Error Reporting API.
+    /// </summary>
+    public sealed class ReportEventsTo
```

Comment:
I'll add an issue for this.  I have something similar to this for the logging portions.